test/rest.test.js

```diff
@@ -3040,4 +3064,8 @@ describe('strong-remoting-rest', function() {
 
     return sharedClass.defineMethod(methodName, options);
   }
+
+  function unhandledRejection(err) {
+    throw new Error('Unhandled promise rejection found in test', err);
```

Comment:
This produces error message that's not very helpful:

```
   Uncaught Error: Unhandled promise rejection found in test
      at process.unhandledRejection (test/rest.test.js:3069:11)
      at processEmit [as emit] (node_modules/nyc/node_modules/signal-exit/index.js:155:32)
      at emitPendingUnhandledRejections (internal/process/promises.js:94:22)
      at process._tickCallback (internal/process/next_tick.js:189:7)
```

Let's throw the original error instead, here is what I am seeing with the change in place:

```
    Uncaught Error: Can't set headers after they are sent.
      at validateHeader (_http_outgoing.js:494:11)
      at ServerResponse.setHeader (_http_outgoing.js:501:3)
      at ServerResponse.header (node_modules/express/lib/response.js:767:10)
      at HttpContext.done (lib/http-context.js:84:475)
      at lib/rest-adapter.js:84:1395
      at node_modules/async/dist/async.js:3874:9
      [etc.]
```